src/rename.py

```diff
@@ -46,5 +44,5 @@ def on_accept(self, _widget, parent, item):
             parent.plot_settings.right_label = text
         if item == parent.canvas.title:
             parent.plot_settings.title = text
-        graphs.reload(parent)
+        parent.canvas.set_axis_properties()
```

Comment:
The actual titles are not renamed until the graph is completely reloaded. So at the moment changing the graph titles does not work in this PR.

Changing the title, then resizing the window works, the problem lies in the lack of reloading..